src/components/NewMessage/NewMessageAbsenceInfo.vue

```diff
@@ -15,6 +15,13 @@
 				disable-tooltip />
 		</template>
 		<h4 class="absence-reminder__caption">{{ userAbsenceCaption }}</h4>
+		<div v-if="hasReplacementUser" class="absence-reminder__replacement">
```

Comment:
We could avoid creating a computed property here, if it's just about checking for truthy value
```suggestion
		<div v-if="userAbsence.replacementUserId" class="absence-reminder__replacement">
```